aries-site/src/themes/aries.js

```diff
@@ -4,16 +4,19 @@ import { deepMerge, normalizeColor } from 'grommet/utils';
 export const aries = deepMerge(hpe, {
   defaultMode: 'dark',
   formField: {
-    border: {
-      error: {
-        color: 'text-weak',
-      },
-    },
-    error: {
+    disabled: {
       background: {
-        color: { light: '#FF40404D', dark: '#FF404066' },
+        color: undefined,
```

Comment:
do we need `undefined` ? Since these are in the stable branch theme the colors are already defined

Reply:
I don't believe the disabled state for FormFields have background colors in the design, however, base.js and hpe theme currently have a background color so something needs to overwrite it. I could be totally wrong here though. If wrong, we can just take it out. If correct, I believe this would need to make its way into the hpe theme.

Reply:
I was thinking you replaced undefined to go with the error background color you are right it should be undefined.